planet_explorer/tests/Data/test_add_to_map/planet_orders/5c9e6c59-eb35-485d-ab7d-04a75e9e0f14/Add_to_map_test_QGIS/files/PSScene/20221221_084022_18_2414/20221221_084022_18_2414_metadata.json

```diff
@@ -0,0 +1 @@
+{"id":"20221221_084022_18_2414","type":"Feature","geometry":{"coordinates":[[[18.28138735653141,-33.75207489835009],[18.235718215986875,-33.930909459887886],[18.595039806750812,-33.994401090235954],[18.6401484699692,-33.81544428639961],[18.28138735653141,-33.75207489835009]]],"type":"Polygon"},"properties":{"acquired":"2022-12-21T08:40:22.180138Z","anomalous_pixels":0,"clear_confidence_percent":93,"clear_percent":100,"cloud_cover":0,"cloud_percent":0,"ground_control":true,"gsd":3.8,"heavy_haze_percent":0,"instrument":"PSB.SD","item_type":"PSScene","light_haze_percent":0,"pixel_resolution":3,"provider":"planetscope","ps4b_geometry":{"coordinates":[[[18.28138735653141,-33.75207489835009],[18.235718215986875,-33.930909459887886],[18.595039806750812,-33.994401090235954],[18.6401484699692,-33.81544428639961],[18.28138735653141,-33.75207489835009]]],"type":"Polygon"},"published":"2022-12-21T21:04:58Z","publishing_stage":"finalized","quality_category":"standard","satellite_azimuth":285.1,"satellite_id":"2414","shadow_percent":0,"snow_ice_percent":0,"strip_id":"6158805","sun_azimuth":77.2,"sun_elevation":61,"updated":"2022-12-22T04:42:19Z","view_angle":1.2,"visible_confidence_percent":73,"visible_percent":100}}
```

Comment:
We need to rename the `Data`  to `data` to adhere with the naming scheme used in the plugin.